linux-audio/fftw3f.json

```diff
@@ -35,7 +35,7 @@
         "/lib/cmake",
         "/lib/pkgconfig",
         "/share/man",
-        "*.la",
-        "*.so"
```

Comment:
we remove the .so because these are only use for `-l` by the linker. It's always the somajor it points to that's used.

Reply:
freedesktop-sdk also removes all .so from Platform.

Reply:
> these are only use for -l by the linker

Ah yea that makes sense, for a moment I thought cleanup would remove it before finishing hence something linking against it won't find.